tests/jobQueue/v1/getBulkUpsertStatus.js

```diff
@@ -171,13 +173,13 @@ describe('tests/jobQueue/v1/getBulkUpsertStatus.js, ' +
     });
   });
 
-  it('Even when hundreds of upserts fail, the valid upsert should ' +
+  it('Even when lots of upserts fail, the valid upsert should ' +
```

Comment:
Yeah, I guess the whole point is we can't afford to have it be flappy, so I guess it's more important to err on the side of slower but solid.